svc/pkg/job-log/ops/read/src/lib.rs

```diff
@@ -3,6 +3,7 @@ use rivet_operation::prelude::*;
 
 #[derive(clickhouse::Row, serde::Deserialize)]
 struct LogEntry {
+	// In nanoseconds
 	ts: i64,
```

Comment:
The current nanoseconds since epoch (~1717623000000000000) is approaching the i64 max (9223372036854775807), both are 1x10^18. Will this pose risk of overflow at all?

Reply:
Corresponds to the year 2262, ill let the rivet engineers of that era deal with it

Reply:
lmao true, didn't think there would be any math that could cause it but just wanted to check :P